### PR TITLE
fist story complete

### PR DESCRIPTION
**Pull request by MikeSC27**

### CODE REVIEW

File: app/controllers/animals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    def show
+        animal = Animal.find(params[:id])
+        render json: animal
+    end
+    def create
+        animal = Animal.create(animal_params)
+        if animal.valid?
+            render json: animal
+        else
+        render json: animal.errors
```
Comment (WalkerTexas-coder) [inline app/controllers/animals_controller.rb]: indentation

File: app/controllers/animals_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        end
+    end
+    def update
+        animal = Animal.find(params[:id])
+        animal.update(animal_params)
+        if animal.valid?
+            render json: animal
+        else
+        render json: animal.errors
```
Comment (WalkerTexas-coder) [inline app/controllers/animals_controller.rb]: indentation

File: app/controllers/animals_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        end
+    end
+    def destroy
+        animal = Animal.find(params[:id])
+        if animal.destroy
+            render json: animal
+        else
+        render json: animal.errors
```
Comment (WalkerTexas-coder) [inline app/controllers/animals_controller.rb]: indentation

**Review by WalkerTexas-coder — APPROVED** [review]
Indentation comments and approval

File: app/controllers/animals_controller.rb (hunk 2 above)
Comment (MikeSC27) [reply app/controllers/animals_controller.rb]: I guess you caught me copy and pasting :) with the same indentation

**Pull request merged**